jobs/email/ReviewReminder.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * @file jobs/email/ReviewReminder.php
+ *
+ * Copyright (c) 2014-2024 Simon Fraser University
+ * Copyright (c) 2000-2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ReviewReminder
+ *
+ * @ingroup jobs
+ *
+ * @brief Class to handle a job to send an review reminder
+ */
+
+namespace PKP\jobs\email;
+
+use APP\facades\Repo;
+use APP\core\Services;
+use Illuminate\Support\Facades\Mail;
+use PKP\log\event\PKPSubmissionEventLogEntry;
+use PKP\core\PKPApplication;
+use PKP\core\Core;
+use PKP\invitation\invitations\ReviewerAccessInvite;
+use PKP\mail\mailables\ReviewResponseRemindAuto;
+use PKP\mail\mailables\ReviewRemindAuto;
+use PKP\jobs\BaseJob;
+
+
+class ReviewReminder extends BaseJob
+{
+    public function __construct(
+        public int $reviewAssignmentId,
+        public int $submissionId,
+        public int $contextId,
+        public string $mailableClass
+    )
+    {
+        parent::__construct();
+
+        $this->reviewAssignmentId = $reviewAssignmentId;
```

Comment:
Since you're using promotion, there's no need to assign these in the constructor.